storage/integration_test.go

```diff
@@ -3417,6 +3417,7 @@ func TestIntegration_UpdateRetentionExpirationTime(t *testing.T) {
 	h.mustWrite(obj.NewWriter(ctx), randomContents())
 
 	defer func() {
+		t.Helper()
```

Comment:
What effect does this have?

Reply:
Helper marks the calling function as a test helper function. When printing file and line information, that function will be skipped. Basically it replaces `loc()` here  - I removed that now too